addons/web_editor/static/src/js/wysiwyg/PeerToPeer.js

```diff
@@ -563,6 +571,8 @@ export class PeerToPeer {
         }
         const clientInfos = this.clientsInfos[clientId];
         if (!clientInfos || clientInfos.fallbackTimeout) return;
+        const backoffFactor = this.clientsInfos[clientId].backoffFactor;
+        const backoffDelay = delay * Math.pow(2, backoffFactor);
```

Comment:
Unless I misunderstood something, this is not the delay to backoff, this is the delay to retry.